sdk/resourcemanager/resources/armresources/ze_generated_example_providers_client_test.go

```diff
@@ -54,7 +54,6 @@ func ExampleProvidersClient_NewListPager() {
 		nextResult, err := pager.NextPage(ctx)
 		if err != nil {
 			log.Fatalf("failed to advance page: %v", err)
-			return
 		}
 		for _, v := range nextResult.Value {
 			// TODO: use page item
```

Comment:
Will you be adding validation of the action and return types?

Reply:
For example, no. We do validation in mock test. I explained it with the mail reply.